packages/sanity/src/structure/i18n/resources.ts

```diff
@@ -385,6 +385,10 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   /** The text for the "Open preview" action for a document */
   'production-preview.menu-item.title': 'Open preview',
 
+  /** Label for button when status is saved */
+  'status-bar.document-status-pulse.status.saved.text': 'Savedblarg',
+  /** Label for button when status is syncing */
+  'status-bar.document-status-pulse.status.syncing.text': 'Saving...blarg',
```

Comment:
are these intentional? looks like some testing edits snuck into the commit :)

Reply:
whoops! I thought I hit save 😬 to remove those